### PR TITLE
Fix/dependency updates

### PR DESCRIPTION
**Pull request by tejohnso**
## Description
Merges suggested dependabot updates and fixes an issue with modern electron and `got`. 

## Motivation and Context
[Upgrade Windows Player](https://docs.google.com/document/d/10Fb_TON_NqbiPa6pP6k1oT8c5drkQYREHqvH_nqVfb4/edit#heading=h.gx4716chb3gk)

## How Has This Been Tested?
Updated unit and integration tests

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: network.js [hunk 1]
```diff
@@ -108,11 +108,10 @@ module.exports = {
 
     function tryDownload(downloadUrl) {
       var file = fs.createWriteStream(savePath);
-      var opts = url.parse(downloadUrl);
```
Comment (tejohnso) [inline network.js:111]: `got` does this internally now

File: test/integration/network.js [hunk 2]
```diff
@@ -95,11 +95,11 @@ describe("Network", function() {
   });
 
   it("gets invalid json message", ()=>{
-    return network.httpFetch("www.google.com.br")
+    return network.httpFetch("http://www.google.com.br")
       .then((resp)=>{
         return resp.json();
       }).catch((error)=>{
-         assert.equal(error.message, "invalid json response body at www.google.com.br reason: Unexpected token < in JSON at position 0");
+         assert(error.message.includes, "invalid json response body");
```
Comment (santiagonoguez) [inline test/integration/network.js:102]: @tejohnso shouldn't this be `assert(error.message.includes("invalid json response body"));` ?

Comment (tejohnso) [reply test/integration/network.js]: Definitely. Good :eye: .  See latest commit.

**Review by santiagonoguez — APPROVED** [review]
Looks good to me

**Pull request merged**